src/lib/controllers/configuration-manager/generators/generate-ec-certificate.js

```diff
@@ -0,0 +1,180 @@
+// Original: https://github.com/balena-io/ec-certificate-generator/blob/master/index.js
```

Comment:
TypeScript and drop the definition file.

Reply:
same as above